CHANGELOG.md

```diff
@@ -178,7 +182,7 @@
 
 * Add warning, file download and numbered steps icons that were originally fetched from govuk_frontend_toolkit ([PR #1154](https://github.com/alphagov/govuk_publishing_components/pull/1154))
 
-## 21.4.0
+## 21.4.0
```

Comment:
Space with... another space? 🤯

Reply:
Yep, seen it before. I think it's `&#160;` (aka &nbsp; in the HTML world) instead of `&#32;`. Funny how Markdown is not flexible with headings formatting in this sense.